bin/merge.sh

```diff
@@ -124,38 +126,53 @@ merge_xdev_into_sdev () {
 }
 
 
-check_cv32e40x_repo() {
+clone_x_dv() {
+
+  echo "=== Cloning x-dv ==="
 
-  echo "=== Check if cv32e40x exist ==="
-  if [ ! -d "./cv32e40x/" ]; then
-    echo "Directory cv32e40x does not exists."
-    echo "Run: ./bin/clonetb -x"
-    echo "before running: ./bin/merge_script --s_into_x-dv"
-    exit 1
-  fi
-  printf "OK\n\n"
+  read -p "This overwrites 'cv32e40x/'. Continue? y/n " yn
+  case $yn in
+    [Yy]* ) ;;
+    * ) echo "aborting"; exit;;
+  esac
 
-  echo "=== Check if cv32e40x is the core-v-verif repo ==="
-  if [ ! -d "./cv32e40x/.git/" ]; then
-    echo "Directory cv32e40x is a 'core-v-verif' repo and not a 'cv32e40x-dv' repo."
-    echo "Run: ./bin/clonetb -x"
-    echo "before running: ./bin/merge_script --s_into_x-dv"
-    exit 1
-  fi
-  printf "OK\n\n"
+  ./bin/clonetb --x-main
 
 }
 
 
 check_merge_status() {
+
   git status
+
+}
+
+
+rejection_diff() {
```

Comment:
I realized I wanted to add this one as well, so I don't have to think as much the next time around.

Reply:
That is very nice!